source/_integrations/unifi.markdown

```diff
@@ -112,17 +112,11 @@ Entities appear for each WLAN. Changing the state of WLAN will trigger a reconfi
 
 ## Sensor
 
-These sensors are disabled by default. Enabling a sensor adds a new device to Home Assistant for every connected client in your network. This can add complexity to large networks.
-
-1. To enable the sensors, on the UniFi integration page, select **Configure**. 
-2. Go to page 3/3 and enable the sensors.
-
-Currently, it is not possible to get a global bandwidth sensor for a UDM router. The integration only supports per-device sensors.
-
+These sensors are disabled by default. Enabling a sensor adds a new device to Home Assistant for every connected client in your network. This can add complexity to large networks. The integration only supports per-device sensors. 
```

Comment:
This should also be moved and refined along the lines of my previous suggestion as this doesn't apply to all sensors